### PR TITLE
Add default.properties to store public URL

### PR DESCRIPTION
**Pull request by Vampouille**
and remove occurrence of public URL "georchestra.mydomain.org" in webapps configuration.
Done in : 
* extractorapp
* console

### CODE REVIEW

File: extractorapp/templates/extractor-email-ack-template.tpl [hunk 1]
```diff
@@ -5,4 +5,4 @@ If you have not requested an extraction, please just ignore this message.
 This e-mail is sent automatically, any answer will be ignored.
 
 ---
-Sent by geOrchestra (https://georchestra.mydomain.org/)
+Sent by geOrchestra ({publicUrl})
```
Comment (fvanderbiest) [inline extractorapp/templates/extractor-email-ack-template.tpl]: trailing slash ?

Comment (Vampouille) [reply extractorapp/templates/extractor-email-ack-template.tpl]: No trailling slash but the link is valid. We can also add a slash in template : 
```
Sent by geOrchestra ({publicUrl}/)
```

**Comment by Vampouille** [comment]
To merged after https://github.com/georchestra/georchestra/pull/2033

**Comment by Vampouille** [comment]
Finished and runtime tested, please review :-)

File: security-proxy/security-proxy.properties [hunk 2]
```diff
@@ -1,19 +1,18 @@
 # ------   proxy-servlet.xml   ---------
-public.host=https://georchestra.mydomain.org/
```
Comment (fvanderbiest) [inline security-proxy/security-proxy.properties]: `public.host=${publicUrl}` ?
To make surpercharging it more obvious

**Comment by fvanderbiest** [comment]
LGTM again !

File: security-proxy/security-proxy.properties (hunk 2 above)
Comment (Vampouille) [reply security-proxy/security-proxy.properties]: Done with 97151e0

**Comment by Vampouille** [comment]
Be careful with the merge to docker-master branch. We need to keep this line as is : 
https://github.com/georchestra/datadir/blob/docker-master/security-proxy/security-proxy.properties#L14

**Comment by Vampouille** [comment]
Merge done with: 87880ff

**Pull request merged**